ytgov/travel-authorization
# Re-Enable Safer Initializer Pipeline

# Context

The previous migration pipeline would kill entire app, and cause an infinite restart in Kubernetes. Now it just complains.
I've also added a database health check middleware that returns a 503 on connection failure, rather than killing the entire app.
This should make future debugging production much easier.

Also automated seeding, though this will need to be change in full production mode as current seeding wipes the database on each run.

At some point we should be redirecting the user to at status page or something if the back-end is throwing 500 errors.

# Testing Instructions

1. Create an invalid config in `/api/.env.development` (e.g. rename the `DB_HOST` to `bad-host`)
2. Build the app via `docker compose up --build`
3. Check that the app boots, but throws some errors about the pipeline failing.
4. Check that you can log in, but that you don't get any content, and the network pane throws a lot of 503.
